src/sentry/replays/endpoints/project_replay_summarize_breadcrumbs.py

```diff
@@ -242,10 +256,10 @@ def fetch_feedback_details(feedback_id: str | None, project_id):
                 category="feedback",
                 id=feedback_id,
                 title="User Feedback",
-                timestamp=event.get("timestamp", 0.0) * 1000,  # feedback timestamp is in seconds
+                timestamp=event.get("timestamp", 0.0) * 1000,  # convert to milliseconds
```

Comment:
```suggestion
                timestamp=event["timestamp"] * 1000,  # convert to milliseconds
```

Nit, works cause the new condition below

Reply:
same comment as above - his part is evaluated first so we need to default to 0 for the calculation (None * 1000 will cause an error)

Reply:
list comprehension evaluates the `if` first, 
`list = [c for a if b]` is the same as if you wrote
```
for a:
  if b:
    list.append(c)
```